llvm/lib/Target/AIE/AIE2Schedule.td

```diff
@@ -495,11 +501,36 @@ MemInstrItinData<II_LDA_POST_1D_R,
 	EmptyCycles<4>, SimpleCycle<R_WA_PORT>], [7,1,1,1],
     MemoryCycles<[5]>>,
 MemInstrItinData<II_LDA_POST_2D,
-    [AvoidPartWordStore, EmptyCycles<1>, SimpleCycle<LOAD_UNIT_A>, EmptyCycles<4>,
+    [AvoidPartWordStore, EmptyCycles<1>, SimpleCycle<LOAD_UNIT_A>,
+  	 EmptyCycles<4>,
      PrefixCycle<P_WM_PORT>, PrefixCycle<M_WM_PORT>, PrefixCycle<DJ_WM_PORT>,
      PrefixCycle<DN_WM_PORT>, PrefixCycle<DC_WM_PORT>, SimpleCycle<R_WA_PORT>],
     [7,1,1,1,1],
     MemoryCycles<[5]>>,
+MemInstrItinData<II_LDA_POST_2D_DC,
+    [AvoidPartWordStore, EmptyCycles<1>, SimpleCycle<LOAD_UNIT_A>,
+	EmptyCycles<4>, SimpleCycle<DC_WM_PORT>], [7,1,1,1,1],
+    MemoryCycles<[5]>>,
+MemInstrItinData<II_LDA_POST_2D_DJ,
+    [AvoidPartWordStore, EmptyCycles<1>, SimpleCycle<LOAD_UNIT_A>,
+	EmptyCycles<4>, SimpleCycle<DJ_WM_PORT>], [7,1,1,1,1],
+    MemoryCycles<[5]>>,
+MemInstrItinData<II_LDA_POST_2D_DN,
+    [AvoidPartWordStore, EmptyCycles<1>, SimpleCycle<LOAD_UNIT_A>,
+	EmptyCycles<4>, SimpleCycle<DN_WM_PORT>], [7,1,1,1,1],
+    MemoryCycles<[5]>>,
+MemInstrItinData<II_LDA_POST_2D_M,
+    [AvoidPartWordStore, EmptyCycles<1>, SimpleCycle<LOAD_UNIT_A>,
+	EmptyCycles<4>, SimpleCycle<M_WM_PORT>], [7,1,1,1,1],
+    MemoryCycles<[5]>>,
+MemInstrItinData<II_LDA_POST_2D_P,
+    [AvoidPartWordStore, EmptyCycles<1>, SimpleCycle<LOAD_UNIT_A>,
+	EmptyCycles<4>, PrefixCycle<DC_WM_PORT>, SimpleCycle<P_WM_PORT>], [7,1,1,1,1],
```

Comment:
Why do we book DC and P ports here?

Reply:
(Same for 3D post-inc)

Reply:
Its a mistake, there is no reason to reserve DC port. Will fix it